api/python/t4/packages.py

```diff
@@ -187,28 +187,49 @@ def get(self):
         """
         return _to_singleton(self.physical_keys)
 
-    def deserialize(self):
+    def deserialize(self, deserializer=None):
         """
         Returns the object this entry corresponds to.
 
+        Custom Deserializers:
+            If `deserializer` is given, it is passed the physical key
```

Comment:
Just wondering if passing bytes, rather than the physical key, would be more useful? Of course, reading a file directly can be more efficient than reading it into memory, then parsing - but this requires the user to parse our file:// and s3:// URLs and figure out what to do with them.

Maybe a good solution is to expose `get_bytes`, etc. as an API - `t4.get_bytes` or something?